### PR TITLE
Tj itenerary

### PR DESCRIPTION
**Pull request by archeni**
# Description
I made the parks' info dynamically show on the dom based on what you choose.

Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

# Testing Instructions

chose a park from the dropdown. They all should work.

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings or errors
- [ ] I have added test instructions that prove my fix is effective or that my feature works

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,56 +6,52 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Jake's Itinerary</title>
 
-    <link rel="stylesheet" href="/styles/main.css">
-    <link rel="icon" type="image/png" href="images/favicon1.png">
-</head>
+    <link rel="stylesheet" href="/styles/main.css" />
```
Comment (JSchugt) [inline index.html:9]: The closing bracket has an un-needed "/"

**Review by JSchugt — CHANGES_REQUESTED** [review]
Small change, code looks good!

**Review by cculkin8 — APPROVED** [review]
Minty fresh, glad it's working!

File: index.html (hunk 1 above)
Comment (JSchugt) [reply index.html]: Ship it!

**Review by JSchugt — APPROVED** [review]

**Pull request merged**